modules/data.atmosphere/R/download.MERRA.R

```diff
@@ -0,0 +1,245 @@
+#' Download MERRA data
+#'
+#' @inheritParams download.CRUNCEP
+#' @param ... Not used -- silently soak up extra arguments from `convert.input`, etc.
+#' @return `data.frame` of meteorology data metadata
+#' @author Alexey Shiklomanov
+#' @export
+download.MERRA <- function(outfolder, start_date, end_date,
+                           lat.in, lon.in,
+                           overwrite = FALSE,
+                           verbose = FALSE,
+                           ...) {
+
+  dates <- seq.Date(as.Date(start_date), as.Date(end_date), "1 day")
+
+  dir.create(outfolder, showWarnings = FALSE, recursive = TRUE)
+
+  # Download all MERRA files first. This skips files that have already been downloaded.
+  for (i in seq_along(dates)) {
+    date <- dates[[i]]
+    PEcAn.logger::logger.debug(paste0(
+      "Downloading ", as.character(date), " (", i, " of ", length(dates), ")"
+    ))
+    get_merra_date(date, lat.in, lon.in, outfolder)
+  }
+
+  # Now, post-process
+  start_year <- lubridate::year(start_date)
+  end_year <- lubridate::year(end_date)
+  ylist <- seq(start_year, end_year)
+
+  nyear <- length(ylist)
+  results <- data.frame(
+    file = character(nyear),
+    host = "",
+    mimimetype = "",
```

Comment:
Typo on mimetype

Reply:
Good catch. Fixed in 9bdfd741c. 

The fact that this didn't throw an error is interesting -- I guess the later call to `results$mimetype[i] <- ...` (spelled correctly) means that R was quietly adding the column.